### PR TITLE
Updated GoBuster Guide

### PR DESCRIPTION
**Pull request by TanvirRahmanTaiyeb**

### CODE REVIEW

**Review by cjboyd1999 — CHANGES_REQUESTED** [review]
Changes look fine, my suggestion is to add the following:

- Regarding step 2, where can a wordlist file be found? Add an example such as `/usr/share/wordlists/dirbuster/directory-list-2.3-medium.txt`
- Add a final step, step 3, to simply say "Click Start GoBuster"

**Comment by TanvirRahmanTaiyeb** [comment]
Hi Cam, I have made the necessary changes as per your feedback. Can you please check and approve it?

File: src/components/GobusterTool/Gobuster.tsx [hunk 1]
```diff
@@ -10,7 +10,15 @@ import { LoadingOverlayAndCancelButtonPkexec } from "../OverlayAndCancelButton/O
...
-const description_userguide = "GoBuster is a tool used for directory and file brute-forcing on web servers.";
+const description_userguide = `
+  GoBuster is a tool used for directory and file brute-forcing on web servers.
+  It helps identify hidden files and directories, exposing vulnerabilities
+  such as unauthorized access points or misconfigurations.
+
+  Step 1: Enter a target website URL (e.g., https://example.com).
+  Step 2: Specify the path to a wordlist file (e.g., /usr/share/wordlists/dirbuster/directory-list-2.3-medium.txt).
+  Step 3: Click Start GoBuster to initiate the scan.
+`;
```
Comment (cjboyd1999) [inline src/components/GobusterTool/Gobuster.tsx:21]: The changes you've made should not be in the `const description_userguide` field, please revert this back to what it is in the current `main` branch.

Changes to the steps that are displayed when actually using the tool should be edited in the `const description` field in the `const GoBusterTool = () => {` class. Please see the existing steps on line 49 of your current `Gobuster.tsx` edit.

At the moment, your edits are not reflected in the UI.

**Review by cjboyd1999 — CHANGES_REQUESTED** [review]
Once you've made the changes as noted in my previous comment, please re-commit and I will review again.

**Pull request closed without merging**